extensions/BugModal/web/autosize.min.js

```diff
@@ -1,6 +1,6 @@
 /*!
-	Autosize 3.0.20
+	Autosize 4.0.0
```

Comment:
If you're live-editing .min.js, you could instead commit 4.0.0.min.js *first* unaltered (as its own commit), so that your later commits to fix bug 1380352 include your diff against 4.0.0.min.js.